packages/@ember/-internals/glimmer/lib/helpers/gte.ts

```diff
@@ -0,0 +1,41 @@
+import { Arguments, CapturedArguments, VM } from '@glimmer/runtime';
+import { InternalHelperReference } from '../utils/references';
+
+/**
+@module ember
+*/
+
+/**
+  Compares the two given values with >=
+
+  Example:
+
+  ```handlebars
+  {{gt age 17}}
+
+  {{! be true if `age` is > 17}}
+  ```
+
+  @public
+  @method gte
+  @for Ember.Templates.helpers
+  @since 2.7.0
+*/
+function gte({ positional: { references } }: CapturedArguments) {
+  let left = references[0].value();
+  if (left === undefined || left === null) {
+    return false;
+  }
+  let right = references[1].value();
+  if (right === undefined) {
+    return false;
+  }
+  if (right === null) {
+    return left >= 0;
```

Comment:
This seems odd to me, why do we need to extra logic (vs just using the final `return left >= right`)?

Reply:
Same reason. Typescript doesn't ket me compare things with undefined or null. How can we silence that?